featuretools/tests/utils_tests/test_cli_utils.py

```diff
@@ -28,7 +28,7 @@ def test_installed_packages():
     requirements = ["pandas", "numpy", "tqdm",
                     "PyYAML", "cloudpickle",
                     "dask", "distributed", "psutil",
-                    "Click"]
+                    "click"]
```

Comment:
this should already be fixed on master, if we merge master in it will be resolved